README.md

```diff
@@ -44,6 +44,7 @@ Databases on SQL Server, Azure SQL Database, or Azure SQL Managed Instance which
 
 ### Output Bindings
 - Output bindings against tables with columns of data types `NTEXT`, `TEXT`, or `IMAGE` are not supported and data upserts will fail. These types [will be removed](https://docs.microsoft.com/sql/t-sql/data-types/ntext-text-and-image-transact-sql) in a future version of SQL Server and are not compatible with the `OPENJSON` function used by this Azure Functions binding.
+- Output bindings against tables with columns of data types `DATE`, `DATETIME`, `DATETIME2`, `DATETIMEOFFSET`, or `SMALLDATETIME` will convert those values to ISO8061 format ("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffZ"). This does not happen for functions written in C# out-of-proc or other languages.
```

Comment:
```suggestion
- .NET In-Proc output bindings against tables with columns of data types `DATE`, `DATETIME`, `DATETIME2`, `DATETIMEOFFSET`, or `SMALLDATETIME` will convert values for those columns to ISO8061 format ("yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffZ") before upsertion. This does not happen for functions written in C# out-of-proc or other languages.
```

Is this correct though about only being for .NET In-Proc? I thought upserting always went through the same code path...

Reply:
It is going through the same path but I believe the DateFormatString setting in JsonSerializerSettings only applies to Datetime objects and not the rows coming in as Json from out of proc language workers.
https://github.com/Azure/azure-functions-sql-extension/blob/349eaa77b9059d2809dc6317895993679fe048df/src/SqlAsyncCollector.cs#L420-L424
https://github.com/Azure/azure-functions-sql-extension/blob/349eaa77b9059d2809dc6317895993679fe048df/src/SqlAsyncCollector.cs#L373